disabled/epochNotifier.go

```diff
@@ -5,19 +5,21 @@ import (
 	vmcommon "github.com/ElrondNetwork/elrond-vm-common"
 )
 
-// EpochNotifier is a disabled implementation of EpochNotifier interface
+// EpochNotifier is a disabled implementation of epochNotifier interface
```

Comment:
I think the interface being implemented is vmcommon.EpochNotifier not epochNotifier, so this should be reverted